playbooks/roles/vault-ssh-host/defaults/main.yml

```diff
@@ -0,0 +1,20 @@
+---
+vault_policy_mgmt_token: testtoken
+vault_ssh_signer_ca: ca
+
+vault_host: vault-a-1.net.opencraft.hosting
+
+vault_server_scheme: http
+
+vault_config_dir: "/etc/vault"
+vault_tls_dir: "{{ vault_config_dir }}/tls"
+vault_tcp_tls_cert_file: "{{ vault_tls_dir }}/cert.pem"
+vault_tcp_tls_key_file: "{{ vault_tls_dir }}/key.pem"
+vault_tcp_tls_client_ca_file: "{{ vault_tls_dir }}/ca.pem"
+
+vault_env:
+    VAULT_TOKEN: "{{ vault_policy_mgmt_token }}"
```

Comment:
Similar comment about this token, though we can probably use a less capable policy for this role.

Reply:
it turns out to be needlessly complicated. Instead the CA are stored in files and retrieved when needed.